R/internal.R

```diff
@@ -882,8 +883,7 @@ format.column <- function(x, single.row = FALSE, digits = 2) {
         } else {
             difference <- first.length - first.dot
         }
-        spaces <- paste(rep(" ", difference), collapse="")
-        x[i] <- paste(spaces, x[i], sep="")
+        x[i] <- paste(x[i], sep="")
```

Comment:
You should use `paste0` instead of `paste`, which is just a shortcut for `paste(..., sep = "")`.

Please always put spaces around `=` (though it won't be necessary here anymore).